Makefile

```diff
@@ -78,6 +81,11 @@ ${BUILD_DIR}/build-image-%: ${BUILD_DIR}
 	@if [ "$(*)" = "$(DRUPAL_LATEST)" ]; then \
 	  docker tag $(DESTINATION_DOCKER_IMAGE):$(*) $(DESTINATION_DOCKER_IMAGE):latest; \
 	fi
+	# Extra! Get the bleeding edge one!
+	docker build \
+	  --build-arg DRUPAL_VERSION=$(BLEEDING_EDGE_VERSION) \
+	  --build-arg PHP_VERSION=$(D11_PHP_VERSION) \
+	  -t $(DESTINATION_DOCKER_IMAGE):$(BLEEDING_EDGE_TAG) .
```

Comment:
This works, but re-creates the bleeding edge one once per existing tag, which is suboptimal (even with all layers cached).
Need to learn more about Makefiles to avoid it.

Didn't do extensive testing of the image, but there might be some permissions issues on `settings.php`, and `drush site-install` is not working (might be not _our_ issue, or might be related to the permissions)

Reply:
I think this may run on every version.